cli/create_test.go

```diff
@@ -877,16 +877,16 @@ func TestCreateInvalidKernelParams(t *testing.T) {
 	}
 }
 
-func TestCreateCreatePodPodConfigFail(t *testing.T) {
+func TestCreateCreatePodSandboxConfigFail(t *testing.T) {
```

Comment:
Why is this function name so weird? Is this deliberate ?

Reply:
Certainly not! I guess the last time it was changed is also by some naive script :) Will fix it. Thanks!